eng/pipelines/stress-test-release.yml

```diff
@@ -6,12 +6,9 @@ parameters:
   - name: Environment
     type: string
```

Comment:
It is working like enum.
I think they support enum type.
https://github.com/MicrosoftDocs/azure-devops-docs-pr/blob/master/docs/pipelines/yaml-schema.md#schema-13

Reply:
The docs are kind of confusing, I think. This IS an enum, but you don't declare it by specifying `enum` you declare it by specifying an enum data type which in this case is `string`, and the `values` are the enum variants.

https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#parameters